src/phoneField/PhoneField.style.tsx

```diff
@@ -56,23 +56,26 @@ export const StyledPhoneField = styled.div`
     padding-left: 0 !important;
   }
 
-  &.kirk-error .kirk-phoneField-wrapper--inline {
+  &.kirk-error .kirk-phoneField-wrapper--inline .kirk-phoneField-wrapper--background {
     background: ${color.lightRed};
     animation: phoneFieldError ${transition.duration.fast} ease-in-out;
   }
 
-  &.kirk-error .kirk-selectField,
-  &.kirk-error .kirk-selectField .kirk-icon,
+  &.kirk-error .kirk-selectField-background,
+  &.kirk-error select,
+  &.kirk-error .kirk-selectField-background .kirk-icon,
   &.kirk-error .kirk-textField-wrapper,
-  &.kirk-error .kirk-textField .kirk-textField-wrapper input {
+  &.kirk-error input {
     background: ${color.lightRed};
     border-color: ${color.lightRed};
   }
 
   &.kirk-error .kirk-error-message {
     color: ${color.red};
     display: block;
-    padding: ${space.m};
+    padding-right: calc(${space.m} + ${space.xl});
+    padding-left: calc(${space.m} + ${space.xl});
```

Comment:
In this component `TextField` & `SelectField` already have `space.xl` horizontally, this padding is not provided by the upper wrapper.
So for the `error message` we need to provide this padding + his own padding.